x/mint/keeper/export_test.go

```diff
@@ -27,3 +27,14 @@ func (k Keeper) DistributeToModule(ctx sdk.Context, recipientModule string, mint
 func (k Keeper) DistributeDeveloperRewards(ctx sdk.Context, totalMintedCoin sdk.Coin, developerRewardsProportion sdk.Dec, developerRewardsReceivers []types.WeightedAddress) (sdk.Int, error) {
 	return k.distributeDeveloperRewards(ctx, totalMintedCoin, developerRewardsProportion, developerRewardsReceivers)
 }
+
+// Set the mint hooks. This is used for testing purposes only.
+func (k *Keeper) SetMintHooksUnsafe(h types.MintHooks) *Keeper {
```

Comment:
Note: since thse methods are in `export_test.go`, they do not get compiled into the final binary and are not accessible by the core application logic